vet-service/src/test/java/com/petclinic/vet/presentationlayer/VetControllerIntegrationTest.java

```diff
@@ -29,14 +35,70 @@ class VetControllerIntegrationTest {
     @Autowired
     VetRepository vetRepository;
 
+    @Autowired
+    RatingRepository ratingRepository;
+
 
     Vet vet = buildVet();
     Vet vet2 = buildVet2();
+    Rating rating1 = buildRating("12345", vet.getVetId(), 5.0);
+    Rating rating2 = buildRating("12346", vet.getVetId(), 4.0);
     VetDTO vetDTO = buildVetDTO();
     String VET_ID = vet.getVetId();
     String VET_BILL_ID = vet.getVetBillId();
     String INVALID_VET_ID = "mjbedf";
 
+    @Test
+    void getAllRatingsForAVet_WithValidVetId_ShouldSucceed() {
+        Publisher<Rating> setup = ratingRepository.deleteAll()
+                .thenMany(ratingRepository.save(rating1))
+                .thenMany(ratingRepository.save(rating2));
+
+        StepVerifier
+                .create(setup)
+                .expectNextCount(1)
+                .verifyComplete();
+
+        client
+                .get()
+                .uri("/vets/" + VET_ID + "/ratings")
+                .accept(MediaType.APPLICATION_JSON)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBodyList(RatingResponseDTO.class)
+                .value((list) -> {
+                    assertEquals(2, list.size());
+                    assertEquals(rating1.getRatingId(), list.get(0).getRatingId());
+                    assertEquals(rating1.getVetId(), list.get(0).getVetId());
+                    assertEquals(rating1.getRateScore(), list.get(0).getRateScore());
+                    assertEquals(rating2.getRatingId(), list.get(1).getRatingId());
+                    assertEquals(rating2.getVetId(), list.get(1).getVetId());
+                    assertEquals(rating2.getRateScore(), list.get(1).getRateScore());
+                });
+    }
+
+//    @Test
```

Comment:
Could you explain why this test was added but commented out?

Reply:
It is commented out because I decided to do the input validation as another story.